storage/rocksdb/ha_rocksdb.cc

```diff
@@ -3697,6 +3715,12 @@ static int rocksdb_init_func(void* const p)
     mysql_mutex_unlock(&rdb_sysvars_mutex);
   }
 
+  if (rocksdb_persistent_cache_size > 0) {
```

Comment:
Yes, that is fine since it looks like RocksDB checks for empty string and invalidates. It wasn't clear what the default behavior was.